frame/airdrop/src/lib.rs

```diff
@@ -0,0 +1,1064 @@
+#![doc = include_str!("../README.md")]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+
+pub mod models;
+pub mod weights;
+
+#[cfg(any(feature = "runtime-benchmarks", test))]
+mod benchmarking;
+mod mocks;
+#[cfg(test)]
+mod tests;
+
+#[frame_support::pallet]
```

Comment:
![image](https://user-images.githubusercontent.com/96540347/174063683-a47be248-1ba2-4354-afe7-209608cc78f5.png)
Don't know if it's relevant. But something the rust-analyzer complained about.

Reply:
This is a result of issues we're facing with the patches in our `Cargo.lock` not getting along with rust-analyzer